packages/loading-skeleton/docs/LoadingGraphic.stories.tsx

```diff
@@ -0,0 +1,106 @@
+import React from "react"
+import { Story } from "@storybook/react"
+import { withDesign } from "storybook-addon-designs"
+import { HumanityAtWork, Informative } from "@kaizen/draft-illustration"
+import { Icon } from "@kaizen/component-library"
+import configureIcon from "@kaizen/component-library/icons/configure.icon.svg"
+import { Avatar } from "@kaizen/draft-avatar"
+import { CATEGORIES, SUB_CATEGORIES } from "../../../storybook/constants"
+import { LoadingGraphic } from "../src/LoadingSkeleton"
+
+import { figmaEmbed } from "../../../storybook/helpers"
+import { StoryWrapper } from "../../../storybook/components/StoryWrapper"
+
+export default {
+  title: `${CATEGORIES.components}/${SUB_CATEGORIES.loadingSkeleton}/Loading Graphic`,
+  component: LoadingGraphic,
+  parameters: {
+    chromatic: { disable: false },
```

Comment:
This will enable all the stories (so you wouldn't need to put them on the individual ones). If you want to only send a select few, then you'd want to remove this :)